pkg/apis/apps/v1/defaults.go

```diff
@@ -117,6 +119,22 @@ func SetDefaults_StatefulSet(obj *appsv1.StatefulSet) {
 		*obj.Spec.UpdateStrategy.RollingUpdate.Partition = 0
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.StatefulSetAutoDeletePVC) {
+		if obj.Spec.PersistentVolumeClaimPolicy == nil {
```

Comment:
nit (to avoid repeated constants):

```golang
if obj.Spec.PersistentVolumeClaimPolicy == nil {
  obj.Spec.PersistentVolumeClaimPolicy = &appsv1.StatefulSetPersistentVolumeClaimPolicy{}
}
if len(obj.Spec.PersistentVolumeClaimPolicy.OnSetDeletion) == 0 {
  obj.Spec.PersistentVolumeClaimPolicy.OnSetDeletion = appsv1.RetainPersistentVolumeClaimDeletePolicyType
}
```

Reply:
done